homeassistant/components/plugwise/quality_scale.yaml

```diff
@@ -0,0 +1,65 @@
+rules:
+  ## Bronze
+  config-flow: done
+  test-before-configure: done
+  unique-config-entry: done
+  config-flow-test-coverage: done
+  runtime-data: done
+  test-before-setup: done
+  appropriate-polling: done
+  entity-unique-id: done
+  has-entity-name: done
+  entity-event-setup: done
+  dependency-transparency: done
+  action-setup: done
+  common-modules: done
+  docs-high-level-description: done
+  docs-installation-instructions: done
+  docs-removal-instructions: done
+  docs-actions: done
+  brands: done
+  ## Silver
+  config-entry-unloading: done
+  log-when-unavailable: done
+  entity-unavailable: done
+  action-exceptions: done
+  reauthentication-flow:
+    status: exempt
+    comment: This integration does not have reauthentication (e.g. fixed authentication for Plugwise Smiles)
+  parallel-updates:
+    status: done
+    comment: Using coordinator
+  test-coverage: done
```

Comment:
It would be nice to also have tests that check failing calls in the number tests